packages/mobile/src/app/saga.ts

```diff
@@ -68,6 +78,39 @@ export function* appInit() {
     navigate(Screens.SetClock)
     return
   }
+
+  const allowOtaTranslations = yield select(allowOtaTranslationsSelector)
+  if (allowOtaTranslations) {
+    const lastFetchTime = yield select(otaTranslationsLastUpdateSelector)
+    const timestamp = yield otaClient.getManifestTimestamp()
+    if (lastFetchTime < timestamp) {
+      yield spawn(otaTranslationsSaga)
+    }
+  }
+}
+
+export function* otaTranslationsSaga() {
+  try {
+    const timestamp = yield otaClient.getManifestTimestamp()
```

Comment:
We should prefer `yield call(...)` instead of just `yield something` so we can more easily test this saga.
See https://stackoverflow.com/a/53468319/158525

Reply:
ah yes, good one! updated :)